docs/source/upgrade-guide/index.md

```diff
@@ -146,13 +146,13 @@ further references.
 
 **(Updated: 2021/06/12)** If you have an existing Volto installation and you are using
 listing blocks, you must run an upgrade step in order to match the new listing
-internals. You can find this upgrade step in the `kitconcept.volto` package. You can run
-the step from there if you have installed `kitconcept.volto` in your project, it's named
+internals. You can find this upgrade step in the `plone.volto` package. You can run
+the step from there if you have installed `plone.volto` in your project, it's named
 `Migrate listing blocks from Volto 12 to Volto 13`. You can find it in the `Add-ons`
 control panel. Alternatively, you can transfer it to your own integration packages and
 run it from there.
 
-https://github.com/kitconcept/kitconcept.volto/blob/main/src/kitconcept/volto/upgrades.py#L6-L64
+https://github.com/kitconcept/plone.volto/blob/main/src/kitconcept/volto/upgrades.py#L6-L64
```

Comment:
The route in the URL is wrong, still kitconcept. Also, the comment in the upgrade guide should be amended, but I guess we can do it later.